cli/cmd/install/install.go

```diff
@@ -47,11 +51,13 @@ type InstallSpec struct {
 	dbURI              string
 	version            string
 	enableBackUp       bool
+	enableOtel         bool
 	onlySecrets        bool
 	enablePrometheusHA bool
 	skipWait           bool
 	tsDBTlsCert        []byte
 	tsDBTlsKey         []byte
+	OtelColConfig      string
```

Comment:
nit: Golang convention states that private fields should start with lowercase.
```suggestion
	otelColConfig      string
```